server/services/v1/management.go

```diff
@@ -147,6 +151,71 @@ func (m *managementService) ListNamespaces(ctx context.Context, _ *api.ListNames
 	}, nil
 }
 
+func (m *managementService) getNameSpaceDetails(ctx context.Context) (map[string]map[string]map[string]map[string]string, error) {
+	res := make(map[string]map[string]map[string]map[string]string)
+	tx, err := m.Manager.StartTx(ctx)
+	if err != nil {
+		return nil, errors.Internal("Failed to begin transaction")
+	}
+	namespaces, err := m.TenantManager.ListNamespaces(ctx, tx)
+	if err != nil {
+		_ = tx.Rollback(ctx)
+		return nil, err
+	}
+	_ = tx.Commit(ctx)
+
+	for _, namespace := range namespaces {
+		nsName := namespace.Metadata().Name
+
+		tenant, err := m.TenantManager.GetTenant(ctx, nsName)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, dbName := range tenant.ListDatabases(ctx) {
+			db, err := tenant.GetDatabase(ctx, dbName)
+			if err != nil {
+				return nil, err
+			}
+
+			if db == nil {
+				// database was dropped in the meantime
+				continue
+			}
+
+			for _, coll := range db.ListCollection() {
+				size, err := tenant.CollectionSize(ctx, db, coll)
+				if err != nil {
+					return nil, err
+				}
+				res[nsName][dbName][coll.Name] = map[string]string{
+					"schema": string(coll.Schema),
+					"size":   strconv.FormatInt(size, 10),
+				}
+			}
+		}
+	}
+	return res, nil
+}
+
+func (m *managementService) AllNamespaceDetails(ctx context.Context, _ *api.DetailedNamespaceInfoRequest) (*api.DetailedNamespaceInfoResponse, error) {
+	resMap, err := m.getNameSpaceDetails(ctx)
```

Comment:
if the namespace is passed in the request then just dump that namespace otherwise iterate on all.

Reply:
I just thought we could have DescribeInstance instead.
@pboros @himank what do you think?

Reply:
We are not using Instance anywhere it may be confusing. What about `DescribeCluster`?

Reply:
Actually maybe `DescribeNamespaces` is fine in this context as `DescribeCluster` or `DescribeInstance` may have a lot of other information as well.